spec/classes/init_spec.rb

```diff
@@ -13,6 +13,7 @@
           :concat_basedir => '/doesnotexist',
           :domain         => 'example.org',
           :osfamily       => osfamily,
+          :operatingsystemmajrelease => 7,
```

Comment:
I think it's time to start using https://github.com/mcanevet/rspec-puppet-facts here as well. That should properly test it with string facts.